README.md

```diff
@@ -96,6 +96,8 @@ Multihash filePointer = Multihash.fromBase58("QmPZ9gcCEpqKTo6aq61g2nXGUhM4iCL3ew
 byte[] fileContents = ipfs.cat(filePointer);
 ```
 
+More example usage found [here](https://github.com/ipfs-shipyard/java-ipfs-http-client/tree/master/src/main/java/io/ipfs/api/demo)
```

Comment:
Would a relative link work, in case the repo gets moved again?

Reply:
have made link relative and tested that it works